### PR TITLE
[MIG][14.0]Migrated a fieldservice_timeline module.

### PR DESCRIPTION
**Pull request by AmmarOfficewalaSerpentCS**

### CODE REVIEW

**Comment by AmmarOfficewalaSerpentCS** [comment]
@smangukiya , @murtuzasaleh , @dreispt , @max3903 ,
Can you please review this PR?

File: fieldservice_timeline/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (smangukiya) [inline fieldservice_timeline/__init__.py:2]: @AmmarOfficewalaSerpentCS Correct the copyrights in all files, please?

"# Copyright (C) 2021 Open Source Integrators"
"# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)."

**Review by smangukiya — CHANGES_REQUESTED** [review]
There is orr_disable_autofill/__init__.py file in this PR. Please remove it.

**Review by smangukiya — APPROVED** [review]
Looks good to me.

**Pull request merged**